### PR TITLE
Make name dynamic based on input

### PR DESCRIPTION
**Pull request by fei-example-user**
Modified app to say hello to users by name by first prompting them to enter their name.

### CODE REVIEW

**Comment by michael-fei** [comment]
@fei-example-user: I like this feature. Have you tested it to make sure it works with hyphenated names?

**Comment by fei-example-user** [comment]
@michael-fei: I tested it with hyphenated names and it works.

**Comment by fei-example-user** [comment]
It doesn't currently work with names which have accent symbols over letters. Is support for accent symbols in names required by the client?

**Comment by michael-fei** [comment]
@fei-example-user: Support for accent symbols in names is not required at this time.

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# javascript-hello-world
\ No newline at end of file
+# javascript-hello-world
+
+## Updated
```
Comment (michael-fei) [inline README.md]: Use a better name for this section.

File: main.js [hunk 2]
```diff
@@ -1,4 +1,8 @@
 (function(){
-  let helloString = "Hello, world!";
+  let helloString = "Hello, ";
+  let inputName = prompt("Please enter your name");
```
Comment (michael-fei) [inline main.js]: Add comments explaining how `prompt()` works.

File: README.md (hunk 1 above)
Comment (fei-example-user) [inline README.md]: Okay, I'll use `New Features` instead.

Comment (michael-fei) [inline README.md]: That sounds much better.

**Pull request merged**